infrastructure/postprocess

```diff
@@ -8,3 +8,9 @@ token = JSON.load(json)['token']
 File.open(File.join(tf_dir, 'k8s-sa-token'), 'w', 0o600) do |io|
     io.write token
 end
+File.chmod(0600, File.join(tf_dir, 'k8s-sa-info'))
```

Comment:
Make this a glob and a loop ... just to not have to update the script every time there is a new generated secret file.

Reply:
Ping @sbolivarm